mind-wave.el

```diff
@@ -341,7 +341,7 @@ Then Mind-Wave will start by gdb, please send new issue with `*mind-wave*' buffe
     (define-key map (kbd "C-,")   #'mind-wave-chat-ask-insert-line)
     (define-key map (kbd "C-S-m") #'mind-wave-chat-ask-send-buffer)
     (define-key map (kbd "C-u")   #'mind-wave-chat-continue)
-    (define-key map (kbd "C-i")   #'mind-wave-chat-generate-title)
+    (define-key map (kbd "C-c C-i")   #'mind-wave-chat-generate-title)
```

Comment:
Current binding takes over my tab key. I can remove this line if necessary though.

Reply:
That's ok since mind-wave generate title automatically, this binding is less use.